physionet-django/console/templates/console/generate_reference_email.html

```diff
@@ -4,13 +4,18 @@
   <head>
     <title>Contact reference email</title>
     {% include "base_css.html" %}
+    <style>
+      a {
+	  text-decoration: underline;
+      }
+      </style>
   </head>
   <body>
     <div class="container-fluid">
-      <p>Contact reference</p>
-      <p><a class='btn btn-primary' href="{% url 'complete_credential_applications' %}">Back to manage applications</a></p>
-      <p> {{ application.get_full_name }} -- {{ application.user.email }}</p>
-      <p><a href="{{mailto}}">Contact reference</a></p>
+<!--       <p>Contact reference</p> -->
+      <p style="font-size: .9rem;"><a class="" href="{% url 'complete_credential_applications' %}">Back to pending applications</a></p>
+<!--       <p> Contact reference for {{ application.get_full_name }} -- {{ application.user.email }}</p> -->
```

Comment:
doesn't the text still get displayed here? i'd suggest removing all of the commented sections anyway. no reason to keep them.